pkg/controller/persistentvolume/types.go

```diff
@@ -26,6 +26,14 @@ import (
 )
 
 const (
+	// A PVClaim can request a quality of service tier by adding this annotation.  The value of the annotation
+	// is arbitrary.  The values are pre-defined by a cluster admin and known to users when requesting a QoS.
+	// For example tiers might be gold, silver, and tin and the admin configures what that means for each volume plugin that can provision a volume.
+	qosProvisioningKey = "volume.experimental.kubernetes.io/quality-of-service"
```

Comment:
"qos" as a keyword is used in the context of pods already to mean something else. "storage-class" seems like a better alternative.